src/panel-ladder.tsx

```diff
@@ -0,0 +1,182 @@
+/**
+ * Ladder Panel
+ *
+ * Panel for ladder formats and associated ladder tables.
+ *
+ * @author Guangcong Luo <guangcongluo@gmail.com>
+ * @license AGPLv3
```

Comment:
Let's license this MIT.

```suggestion
 * @license MIT
```

I wrote a lot of stuff about how I want licensing to work, but the short answer is that I prefer to be the only one with AGPL content, so I can relicense it to MIT if someone requests it.